Documentation/op-guide/performance.md

```diff
@@ -10,4 +10,26 @@ There are other sub-systems which impact the overall performance of etcd. Each s
 
 ## Benchmarks
 
-TODO
\ No newline at end of file
+To benchmark etcd performance, simply use [benchmark](https://github.com/coreos/etcd/tree/master/tools/benchmark) CLI tool. Following results are run with:
```

Comment:
Following results are from a 3 member etcd cluster with the hardware configuration: